src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int loadCapacity;
+    private int grossWeight;
```

Comment:
Consider using access modifiers for the `loadCapacity` and `grossWeight` fields. Direct access to class fields is discouraged. They should be private and accessed through getter and setter methods which you've already defined.